### ISSUE TITLE
anaconda not opening in windows 10 using all methods available on google

### ISSUE THREAD

**Issue by vin725k** [ContinuumIO/anaconda-issues#10428]
I am trying to open conda from windows 10 cmd but it always giving it is not recognized as internal or external command
I have tried with 64 bit and my system is 64 bit, anaconda cmd is there but no navigator,

Tried all methoda like conda update but always giving same error.
Please let me know what should  I do

**Comment by SpiceMelange** [comment]
Same issue on 10.0.17763 Build 17763 - KB4471332 (OS Build 17763.194). No version (Anaconda3 or Anaconda2 and x86 or x86_64) will launch the Navigator

**Comment by goanpeca** [comment]
@SpiceMelange are you using the Anaconda Prompt?

**Comment by SpiceMelange** [comment]
I have tried using the Anaconda3 and Anaconda2 prompts, both x86 and x86_64 for both 2 and 3. Obviously, I’ve also tried the Navigators links as well. I also performed a condo update --all, which did not resolve the issue.

Thanks,
Chris Cozad
x4802
Release and Technical Services

**Comment by SpiceMelange** [comment]
From the Anaconda 3 64-bit prompt:

(base) C:\WINDOWS\System32>anaconda-navigator
Warning: QT_DEVICE_PIXEL_RATIO is deprecated. Instead use:
   QT_AUTO_SCREEN_SCALE_FACTOR to enable platform plugin controlled per-screen factors.
   QT_SCREEN_SCALE_FACTORS to set per-screen factors.
   QT_SCALE_FACTOR to set the application global scale factor.
An unexpected error occurred on Navigator start-up<br>'root_prefix'
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\exceptions.py", line 75, in exception_handler
    return_value = func(*args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\app\start.py", line 150, in start_app
    window = run_app(splash)
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\app\start.py", line 65, in run_app
    window = MainWindow(splash=splash, tab_project=False)
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\widgets\main_window.py", line 168, in __init__
    self.api = AnacondaAPI()
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\api\anaconda_api.py", line 2294, in AnacondaAPI
    ANACONDA_API = _AnacondaAPI()
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\api\anaconda_api.py", line 91, in __init__
    self._conda_api = CondaAPI()
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 1767, in CondaAPI
    CONDA_API = _CondaAPI()
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 350, in __init__
    self.set_conda_prefix(info=get_conda_info())
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 505, in set_conda_prefix
    self.ROOT_PREFIX = info['root_prefix']
KeyError: 'root_prefix'

**Comment by mash00d** [comment]
I tried many tricks to install 64 bit version but nothing worked. The installation was working fine for 32 bit but not for 64 bit.

Solution: I had a older version of 64bit anaconda with me, I first installed it then updated the anaconda to latest. Now everything is working fine and I have latest 64 bit anaconda on my windows 10 machine.

The reason why I wanted 64 bit is because there are few packages which don't install on 32 bit instance.
